### PR TITLE
Fix some PHPCS violations

### PR DESCRIPTION
**Pull request by atailouloute**
Hello,
I made some changes in order to follow PSR standards
Kind regards

### CODE REVIEW

File: Index.php [hunk 1]
```diff
@@ -42,15 +43,16 @@ private function createNewFile()
     }
 
     /**
-     * Adds sitemap link to the index file
+     * Adds sitemap link to the index file.
+     *
+     * @param string $location     URL of the sitemap
```
Comment (samdark) [inline Index.php]: It's not part of PSR-2... I don't like such spacing.

File: Index.php [hunk 2]
```diff
@@ -42,15 +43,16 @@ private function createNewFile()
...
+     * @param int    $lastModified unix timestamp of sitemap modification time
      *
-     * @param string $location URL of the sitemap
-     * @param integer $lastModified unix timestamp of sitemap modification time
```
Comment (samdark) [inline Index.php]: integer looks better to me.

File: Sitemap.php [hunk 3]
```diff
@@ -58,10 +59,9 @@ class Sitemap
         self::WEEKLY,
         self::MONTHLY,
         self::YEARLY,
-        self::NEVER
+        self::NEVER,
```
Comment (samdark) [inline Sitemap.php]: It's not part of PSR-2.

File: Sitemap.php [hunk 4]
```diff
@@ -84,11 +85,11 @@ public function __construct($filePath)
     }
 
     /**
-     * Creats new file
+     * Creats new file.
      */
     private function createNewFile()
     {
-        $this->fileCount++;
+        ++$this->fileCount;
```
Comment (samdark) [inline Sitemap.php]: Why?

File: Sitemap.php [hunk 5]
```diff
@@ -202,42 +203,46 @@ private function getCurrentFilePath()
         }
 
         $parts = pathinfo($this->filePath);
-        return $parts['dirname'] . DIRECTORY_SEPARATOR . $parts['filename'] . '_' . $this->fileCount . '.' . $parts['extension'];
+
+        return $parts['dirname'].DIRECTORY_SEPARATOR.$parts['filename'].'_'.$this->fileCount.'.'.$parts['extension'];
```
Comment (samdark) [inline Sitemap.php]: Concatenation isn't covered by PSRs. I'd prefer to have spaces.

File: Sitemap.php [hunk 6]
```diff
@@ -202,42 +203,46 @@ private function getCurrentFilePath()
...
      * Sets number of URLs to be kept in memory before writing it to file.
      * Default is 1000.
      *
-     * @param integer $number
+     * @param int $number
      */
     public function setBufferSize($number)
     {
-        $this->bufferSize = (int)$number;
+        $this->bufferSize = (int) $number;
```
Comment (samdark) [inline Sitemap.php]: Type casting isn't covered by PSR. I'd prefer it to stay as is.

Comment (atailouloute) [inline Sitemap.php]: Actually some of the modifications are made by PHPCS fixer  (https://github.com/FriendsOfPHP/PHP-CS-Fixer) ;

Comment (samdark) [inline Sitemap.php]: But that doesn't mean it's in PSR.

**Comment by samdark** [comment]
I've corrected PSR violations: https://github.com/samdark/sitemap/commit/701a46d605db7aea97f590fbb7f022ee449a5afa

Thanks for pointing them out.

**Pull request closed without merging**